_includes/context/priority.md

```diff
@@ -1,3 +1,97 @@
+<a class="anchor" id="contexttoolbarpriority"></a><a class="anchor" id="contextformpriority"></a>
+
+### Troubleshooting context toolbar and context form conflicts
+
+There are situations where custom context toolbars or custom context forms may conflict with:
+
+  - Context toolbars or context forms provided by the editor,
+  - Context toolbars or context forms provided by the [Quick Toolbars (`quickbars`) plugin]({{site.baseurl}}/plugins/quickbars/),
+  - Other custom context toolbars or custom context forms.
+
+{% if site.productmajorversion < 6 %}
+How these conflicts are resolves depends on the {{site.productname}} version.
+
+- For {{site.productname}} 5.3 or newer, see: [Determining the display priority of context toolbars and context forms]().
+- For {{site.productname}} 5.0, 5.1, or 5.2; see: [Determining the display priority of context toolbars and context forms (Legacy behavior)]().
+{% endif %}
+
+#### Determining the display priority of context toolbars and context forms
+
+There are three settings that determine the priority of context toolbars and context forms: `scope`, `predicate`, and `position`.
+
+- `scope` - Sets the context toolbar or form as visible of certain content (`node`) or a general (or global) toolbar or form (`editor`).
+- `predicate` - Sets a function that tests in the current context position against one or more conditions, returning a boolean value.
+- `position` - Sets where the context toolbar or form is rendered, relative to the current section (`selection`, `node`, or `line`).
+
+Generally:
+
+- Context _forms_ are prioritized over context _toolbars_.
+- Context _forms_ with `scope: 'node'` are prioritized over `scope: 'editor'`.
+- Only one context _form_ can be shown for a selection or cursor position, they cannot be concatenated to another context _form_ or a context _toolbar_.
+- Context _toolbars_ with `position: 'selection'` are prioritized over `position: 'node'`, and `position: 'line'` is given the lowest priority.
+- When the editor concatenate context _toolbars_ where there is more than one context _toolbar_ to display with:
```

Comment:
You've got extra words in this or something

Reply:
not sure what happened there. Fixed?